### DISCUSSION TITLE
Help with Organization's

### DISCUSSION THREAD

**Discussion by BuddiesTV** [community/community#23574] [New to GitHub]
<p>Hello GitHub Community,<br>
I deleted my Organization to clean up every setting, remove every member and delete repositories.<br>
After i deleted it, i wanted to re-create the Organization with the same Name, but i discord says the Name is unavailable “Organization name ‘SycroV’ is unavailable.”<br>
is there a way to create a new Organization with this name after deleting it?</p>
<p>best regards BuddiesTV | Lucas</p>

**Comment by jfullstackdev** [comment]
<p>check this out,</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/organizations/managing-organization-settings/deleting-an-organization-account">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/organizations/managing-organization-settings/deleting-an-organization-account" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/47092464/181077983-754f40c2-4223-4ca3-aa26-755d9463b12e.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/organizations/managing-organization-settings/deleting-an-organization-account" target="_blank" rel="noopener">Deleting an organization account - GitHub Docs</a></h3>

  <p>When you delete an organization, all repositories, forks of private repositories, wikis, issues, pull requests, and Project or Organization Pages are deleted as well. Your billing will end, and after 90 days the organization name becomes available...</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by BuddiesTV** [comment]
<p>Thank you!<br>
But is there no way to speed up this process a little? :smiley:</p>

**Comment by jfullstackdev** [comment]
<p>I’m not very sure and I cannot comment on behalf of GitHub,</p>
<p>what you can do is to request maybe,</p>
<p><a href="https://support.github.com/contact/account">https://support.github.com/contact/account</a></p>

**Comment by BuddiesTV** [comment]
<p>I already replied to the Email, that was send because i deleted the Organization &amp; got a email back with a Ticket id.<br>
But when i click on “My Tickets” it says “You don’t have any tickets yet.”.<br>
Should i create a new ticket on the Support site( <a href="https://support.github.com/contact/account" rel="noopener nofollow ugc">https://support.github.com</a>)?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>yup, if there is really none, you can create one again,</p>

**Comment by BuddiesTV** [comment]
<p>Thank you very much!</p>
